### PR TITLE
[stable/kube-state-metrics] update kube-state-metrics README.md

### PR DESCRIPTION
**Pull request by tariq1890**
Updating what I missed in my PR earlier "https://github.com/helm/charts/pull/11316"

### CODE REVIEW

File: stable/kube-state-metrics/Chart.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ keywords:
 - metric
 - monitoring
 - prometheus
-version: 0.14.0
+version: 0.14.1
```
Comment (cpanato) [inline stable/kube-state-metrics/Chart.yaml:8]: ~would bump to this~
~```suggestion~
~version: 0.15.0~
~```~

Comment (cpanato) [reply stable/kube-state-metrics/Chart.yaml:8]: ignore my comment, saw this is just a bug fix

**Review by cpanato — APPROVED** [review]

**Comment by tariq1890** [comment]
Thanks @cpanato!

**Pull request merged**